src/gateway/kong-production/monitoring/statsd.md

```diff
@@ -1,6 +1,99 @@
 ---
-title: Monitoring with Statsd
-
+title: Monitoring with StatsD
+content_type: how-to
 ---
 
-## PLACEHOLDER How to Monitor with Statsd
\ No newline at end of file
+[StatsD](https://github.com/statsd/statsd) is a network daemon that collects
+and aggreates performance metrics by listening on the network for simple 
+text based statistics data published by applications. 
+
+This guide will help you setup a test {{site.base_gateway}} and
+StatsD service. Then you will generate sample requests to {{site.base_gateway}} and
+observe the collected monitoring data. 
+
+### Prerequisites
+* [Docker](https://docs.docker.com/get-docker/) is installed locally. It is used to run {{site.base_gateway}}, the supporting database, and StatsD locally. 
+
+   If you will be using an existing {{site.base_gateway}} instance instead of the test instance, you may need configure Docker to ensure features like network connectivity function correctly.  
+* [curl](https://curl.se/) is installed locally. It is used to send requests to {{site.base_gateway}}. `curl` is pre-installed on most systems.
+* [Netcat](http://netcat.sourceforge.net/) is installed locally as `nc` on the `PATH`. `nc` is used to send requests 
+  to the StatsD management interface. `nc` is pre-installed on many systems.
+
+### Configure StatsD monitoring
+
+{:.note}
+   > **Note:** This guide's instructions use a test {{site.base_gateway}} instance to demonstrate how the StatsD plugin can be used to collect metrics. If you want to use an existing {{site.base_gateway}} instance for this guide, you must modify the connection information in the commands.
+
+1. Optional: Install {{site.base_gateway}} using the following script:
```

Comment:
@rspurgeon I moved the note and script back to the steps. Does this look good to you?

Reply:
@cloudjumpercat  I've just pushed a revision, please take a look